lib/serialisers/json-0.6.js

```diff
@@ -44,14 +44,30 @@ module.exports = JSONSerialiser.extend({
     } else if (element.content !== undefined) {
       var content = this.serialiseContent(element.content);
 
-      if (content !== undefined) {
+      if (this.shouldSerialiseContent(element, content)) {
         payload['content'] = content;
       }
     }
 
     return payload;
   },
 
+  shouldSerialiseContent: function(element, content) {
+    if (content === undefined) {
+      return false;
+    }
+
+    if (element.element === 'httpRequest' || element.element === 'httpResponse' || element.element === 'category') {
```

Comment:
I don't like this line. It's minim-parse-result specific.

Reply:
We already have various logic here that is minim-api-description and minim-parse-result-specific. This is due to Refract 0.6 having element specific serialisation rules which we are moving away from in 1.0.

The whole point of this file is to bridge the gap from the old state to the future(current).

> ```js
> sourceMapSerialiseContent: function(element) {
>   return element.toValue();
> },
> ```

> ```js
> // "meta" attribute has been renamed to metadata
> var metadata = element.attributes.get('meta');
> ```

> ```js
> dataStructureSerialiseContent: function(element) {
>   return [this.serialiseContent(element.content)];
> },
> ```

Reply:
Could the line be more readable as

```js
['httpRequest', 'httpResponse', 'category'].includes(element.element)
```

or even

```js
VERY_SMART_CONSTANT_NAME.includes(element.element)
```

?

Reply:
@honzajavorek We still support Node 4 (and browsers) with no transpilation so we can't use `includes` as it doesn't exist everywhere.